### DISCUSSION TITLE
模块列表显示灰色更新图标，实际上该模块已经是最新版

### DISCUSSION THREAD

**Discussion by httpsColon** [LSPosed/LSPosed#2010] [General]
![8~45)3D3V2OWR7HMZP(9O9N](https://user-images.githubusercontent.com/102579986/175976566-47f77e6e-5b19-4399-a0c4-3c983b5c3007.jpg)
模块列表显示灰色更新图标，实际上该模块已经是最新版

**Comment by yujincheng08** [comment]
然后呢？灰色更新按钮比就是说明已经最新？有什么问题？
